src/index.html

```diff
@@ -7,12 +7,53 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+
     <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
       rel="stylesheet"
-      href="./styles/index.scss"
+    />
+    <link
+      rel="/src/styles/index.scss"
+      href="/src/styles/index.css"
     />
   </head>
+
   <body>
     <h1>Product cards</h1>
```

Comment:
Ensure that the 'h1' tag is being used properly for the page title. If 'Product cards' is the main topic of the page, then it's fine. Otherwise, consider using a more descriptive title or a different tag if it's not the main topic.